components/brave_wallet_ui/components/extension/transaction-box/zcash-transaction-detail-box.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
Lets rename this file to use `snake_case` convention.

`zcash_transaction_detail_box.tsx`